woocommerce/woocommerce-android
# Store creation: Set "coming soon" as default status

This PR updates the new site status after store creation to `COMING SOON`, which brings it on par with the web. Previously, the sites were created as `PRIVATE`.

**To test:**
1. Go to the site picker and trigger the site creation flow (make sure you have some free credit)
2. After the site is created, go to the web and verify it has the "coming soon" tag next to its name, like this:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/1522856/226037566-0aabada1-136c-4099-a1b0-8b2f55259d17.png">
